packages/react-components/src/atoms/Toggle.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import css from '@emotion/css';
+import { lineHeight, perRem } from '../pixels';
+import { steel, paper, tin, charcoal } from '../colors';
+const checkboxLabelStyles = css({
+  display: 'flex',
+  height: '54px',
+  borderRadius: '27px',
+  border: `1px solid ${steel.rgb}`,
+  overflow: 'hidden',
+  backgroundColor: steel.rgb,
+  color: tin.rgb,
+  g: {
+    stroke: tin.rgb,
+  },
+});
+
+const buttonStyle = css({
+  width: '50%',
+  borderRadius: '27px',
+  display: 'flex',
+  justifyContent: 'center',
+  alignItems: 'center',
+});
+
+const iconStyles = css({
+  display: 'inline-block',
+  width: `${lineHeight / perRem}em`,
+  height: `${lineHeight / perRem}em`,
+  paddingRight: `${6 / perRem}em`,
+});
+const activeStyles = css({
+  backgroundColor: paper.rgb,
+  color: charcoal.rgb,
+  g: {
+    stroke: charcoal.rgb,
+  },
+});
+const checkboxStyles = css({
+  display: 'none',
+  '&:not(:checked) ~ .left': activeStyles,
+  '&:checked ~ .right': activeStyles,
+});
+const buttonRight = css({ marginLeft: '-20px', padding: '0 10px' });
+const buttonLeft = css({ marginRight: '-20px', padding: '0 10px' });
+
+interface ToggleProps {
+  defaultChecked?: boolean;
+  leftButtonText: string;
+  leftButtonIcon: React.ReactElement<SVGElement>;
+  rightButtonText: string;
+  rightButtonIcon: React.ReactElement<SVGElement>;
+  onChange: (e: React.FormEvent<HTMLInputElement>) => void;
```

Comment:
Related to the interface definition, can we not expose the whole event? The component that receives this should not care about `target` etc., and giving them just the new state is the simplest in usage as well